tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -754,7 +754,13 @@ module ProjectOperations =
             |> Seq.toArray
 
         Assert.Equal<(string * int * int * int)[]>(expected |> Seq.sort |> Seq.toArray, actual)
-
+        
+    let expectNone x =
+        if Option.isSome x then failwith "expected None, but was Some"
+    
+    let expectSome x =
```

Comment:
We could make some stronger assertions here. We could save the results we get when we do `checkFile` into the `WorkflowContext` and then check we're getting back the same thing. Or even just reuse the signatures that are already there and we can generate from the check result. We'd just have to use `updatePublicSurface` instead of `updateInternal` for the changes. Not necessary though, feel free to leave it as it is.

Reply:
If you're okay with it, I'd like to do it in a separate PR.
That would help us with moving the whole story forward a bit quicker, I think.

Reply:
Okay, seemed small enough to squeeze in here 😁